services/forms-api/lambdas/getForm.js

```diff
@@ -64,7 +65,7 @@ function formQueryToObj(items) {
      * If the item exsists and have children add them to the item.
      */
     const itemSK = hashArray.join('#');
-    let item = omitObjectKeys(findItemInQuery(items, 'SK', itemSK), ['SK', 'PK']);
+    let item = objectWithoutProperties(findItemInQuery(items, 'SK', itemSK), ['SK', 'PK']);
```

Comment:
If we don't are interested in the values of updatedAt and createdAt we could remove them here already (y)